scripts-available/CDB_PG_Federated_Tables.sql

```diff
@@ -0,0 +1,295 @@
+----------------------------------------------------------------------
+-- Federated Tables management functions
+----------------------------------------------------------------------
+
+-- Take a config jsonb and transform it to an input suitable for
+-- _CDB_SetUp_User_PG_FDW_Server
+CREATE OR REPLACE FUNCTION @extschema@.__ft_credentials_to_user_mapping(input_config jsonb)
```

Comment:
What does ft mean?

Reply:
Same question for the other functions. Whatever that is, is it necessary?

Reply:
`__ft_*` stands for "federated table" and I'm copying that pattern from good old C, as poor man's way of keeping private functions in their own "namespace"